docs/book/src/reference/versions.md

```diff
@@ -52,7 +52,10 @@ These diagrams show the relationships between components in a Cluster API releas
 | Kubernetes v1.20 | ✓                                | ✓                             | ✓                                | ✓                             | ✓                                | ✓                             |
 | Kubernetes v1.21 | ✓                                | ✓                             | ✓                                | ✓                             | ✓                                | ✓                             |
 | Kubernetes v1.22 |                                  | ✓                             | ✓                                | ✓                             | ✓                                | ✓                             |
-| Kubernetes v1.23 |                                  |                               | ✓                                | ✓                             | ✓                                | ✓                             |
+| Kubernetes v1.23* |                                  |                               | ✓                                | ✓                             | ✓                                | ✓                             |
+
+\* There is an issue with CRDs in Kubernetes v1.23.{0-2}. ClusterClass with patches is affected by that (for more details please see [this issue](https://github.com/kubernetes-sigs/cluster-api/issues/5990)). Therefore we recommend to use Kubernetes v1.23.3+ with ClusterClass.
```

Comment:
Q: should the the note go at the end of the paragraph

Reply:
Also, should the header of the table have CAPI v1beta1 (v1.0) --> CAPI v1beta1 (v1.{0,1}) (or v1.x)
Not strictly related to this PR, but useful for the release...

Reply:
> Q: should the the note go at the end of the paragraph

I assume you mean below l.60? I though it's good to have it directly under the table as the table is referring to it.
But I'm fine to change it if you prefer having it below.

You're right about the table header. I'll probably change it as suggested if I can't come up with something better :)

While it's great to have this overview, I wonder if it's confusing to users if they see a different version of this table in every book (and I assume some have conflicting information as we don't really maintain old versions).
So we  maybe should consider some changes (like e.g. the book of each version only contains the information which Kubernetes versions are supported in that release). 
But as that's probably not a quick discussion, let's ignore that for now.

Reply:
I went with 1.x and also adjusted some occurrences in the text above.

Let me know if you would prefer having the note further down